meson-scripts/gen_build.in.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * NOTE: THIS FILE IS AUTO-GENERATED BY THE BUILD SYSTEM AND IS NOT TRACKED BY
+ * SOURCE CONTROL. DO NOT EDIT THIS FILE -- IT WILL BE OVERWRITTEN BY
+ * SUBSEQUENT BUILDS.
+ */
```

Comment:
Reviewers: This is kind of confusing because _this_ file (`gen_build.in.h`) of course is actually tracked by source control, but the `build.h` file it generates by replacing `@VCS_TAG` is not tracked. We could get around this by echoing this comment to the `build.h` file from `gen_bindings_h`, but it seemed better to have less logic in bash given that this file is basically never going to change. I'm fine with either way.